accounts-db/src/accounts_index/in_mem_accounts_index.rs

```diff
@@ -162,9 +162,11 @@ struct StartupInfo<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> {
 
 #[derive(Default, Debug)]
 /// result from scanning in-mem index during flush
-struct FlushScanResult<T> {
+struct FlushScanResult {
     /// pubkeys whose age indicates they may be evicted now, pending further checks.
-    evictions_age_possible: Vec<(Pubkey, Arc<AccountMapEntry<T>>)>,
+    /// Each entry contains: (pubkey, is_dirty)
+    /// Entries with ref_count != 1 are filtered out during scan
+    evictions_age_possible: Vec<(Pubkey, bool)>,
```

Comment:
Another common approach is to document the type inline:


```suggestion
    evictions_age_possible: Vec<(Pubkey, /*is dirty*/ bool)>,
```

No strong preference on my side. The inline approach lets you remove that one doc line though.

Reply:
done.